config/example/node.env

```diff
@@ -42,6 +42,11 @@ NEOFS_GRPC_1_ENDPOINT=s02.neofs.devenv:8080
 ### TLS config
 NEOFS_GRPC_1_TLS_ENABLED=false
 
+## 2 server
+NEOFS_GRPC_2_ENDPOINT=s03.neofs.devenv:8080
+### TLS config
+NEOFS_GRPC_2_TLS_ENABLED=true
```

Comment:
What key/cert is used for it?

Reply:
well, none, they are for config reading tests only. [json](https://github.com/nspcc-dev/neofs-node/blob/98f0918b074aa9e53b7c67354cadefd0257f71f5/config/example/node.json#L54-L59) and [yml](https://github.com/nspcc-dev/neofs-node/blob/98f0918b074aa9e53b7c67354cadefd0257f71f5/config/example/node.yaml#L47-L49) are the same. env file was just not tested. do you want to make it false then?

Reply:
It's an example, it should have something more or less realistic, that's the only thing I care about wrt this question.

Reply:
fixed